scripts/AudioManager.js

```diff
@@ -83,16 +83,41 @@ function AudioManager(addEventListener, isTownTune) {
 					audio.volume -= step;
 				} else {
 					clearInterval(fade);
-					audio.pause();
+					playPause(false);
 					audio.volume = oldVolume;
 					if (callback) callback();
 				}
 			}, 100);
 		}
 	}
+	
+	function playPause(play) {
```

Comment:
That was leftovers from when I was doing things at the same time as playing or pausing, instead of using the listeners